ansible/roles/fabric-console/tasks/k8s/create.yml

```diff
@@ -71,6 +71,7 @@
           kubernetes.io/ingress.class: "{{console_ingress_class}}"
           nginx.ingress.kubernetes.io/proxy-connect-timeout: 60s
           nginx.ingress.kubernetes.io/ssl-passthrough: "true"
```

Comment:
ssl-passthrough should be false or altogether removed from here

Reply:
I have set it to false